ngraph/test/runtime/ie/unit_test.manifest

```diff
@@ -1160,40 +1160,21 @@ IE_CPU.onnx_model_reduce_sum_13_axes_as_0_dim_input
 # output mismatch
 IE_CPU.gather_nd_batch_1d_from_3d_negative
 
-# DFT is not implemented yet in plugins
-IE_CPU.dft1d_eval
-IE_CPU.dft2d_eval
-IE_CPU.dft3d_eval
-IE_CPU.dft1d_eval_i32
-IE_CPU.dft2d_eval_i32
-IE_CPU.dft3d_eval_i32
-IE_CPU.dft1d_signal_size_eval
+# CPU plugin does not support float16 and bfloat16 in DFT
```

Comment:
CPU Plugin supports fp16 and bf16 formats. These should be deleted.

Reply:
We need fix only this comment.

Reply:
Comment was fixed.